src/GitVersion.Core/Git/ReferenceName.cs

```diff
@@ -76,6 +78,38 @@ public static ReferenceName FromBranchName(string branchName)
     public override int GetHashCode() => equalityHelper.GetHashCode(this);
     public override string ToString() => Friendly;
 
+    public bool TryGetSemanticVersion([NotNullWhen(true)] out (SemanticVersion Value, string? Name) result,
+        Regex versionPatternRegex, string? labelPrefix, SemanticVersionFormat format)
```

Comment:
When a method signature is so long it needs to be broken up, I prefer that every parameter is given its own line.

```suggestion
    public bool TryGetSemanticVersion([NotNullWhen(true)] out (SemanticVersion Value, string? Name) result,
                                      Regex versionPatternRegex,
                                      string? labelPrefix,
                                      SemanticVersionFormat format)
```

Reply:
done